images/Dockerfile-clientserver

```diff
@@ -53,69 +53,37 @@ RUN mkdir -p /var/www/html/clients/darwin && \
     mkdir -p /var/www/html/clients/linux && \
     mkdir -p /var/www/html/clients/windows
```

Comment:
I realized we could remove the mkdir (since COPY will do it for you), but that does change the final owner of `/var/www/html/clients/*` from `default:root` to `root:root`, so I decided to keep the mkdir just in (the unlikely I guess) case that the owner changes impacts anything.